enable/trait_defs/ui/kiva_font_editor.py

```diff
@@ -0,0 +1,128 @@
+# (C) Copyright 2005-2022 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from pyface.font import Font as PyfaceFont
+from pyface.font_dialog import get_font
+from traits.api import Bool, Callable, Instance, Str, observe
+from traits.trait_base import SequenceTypes
```

Comment:
Possibly not for this PR, but it would be good to move away from this use of `SequenceTypes`. Like anything else not in one of the Traits `*.api` modules, it's not a supported part of the Traits API and may move or disappear in a future Traits version.

Actually, I'm not sure it's even useful here; see question below.

Reply:
Agreed.   I'd like a better way to express the listy/tuply-sort of things consistently and in one place - we already have some definitintions that aren't common which differ eg. on the status of numpy arrays.  There was also some reason why `collections.abc.Sequence` didn't work for this purpose.